packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -409,28 +409,206 @@ StructuredListMediaCalendar.argTypes = {
+      content: `<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'>
+                  <cbp-flex
+                  sx='{
+                        "width":"6.25rem",
+                        "height":"6.25rem",
+                        "display":"flex",
+                        "flex-direction":"column",
+                        "align-items":"center"
+                      }'
+                >
+                  <cbp-flex-item
+                  sx='{
+                        "width":"100%",
+                        "text-align":"center",
+                        "border-top-left-radius":"var(--cbp-border-radius-soft)",
+                        "border-top-right-radius":"var(--cbp-border-radius-soft)",
+                        "padding-block":"var(--cbp-space-2x)",
+                        "background-color":"var(--cbp-color-red-50)",
+                        "border":"1px solid var(--cbp-color-red-50)",
+                        "color":"var(--cbp-color-white)",
+                        "text-transform":"uppercase",
+                        "font-size":"var(--cbp-font-size-heading-xl)",
+                        "font-weight":"var(--cbp-font-weight-regular)"
+                      }'
+                  > 
+                    apr
+                  </cbp-flex-item>
+                  
+                  <cbp-flex-item
+                    sx='{
+                          "width":"100%",
+                          "height":"4.25rem",
+                          "text-align":"center",
+                          "border-bottom-left-radius":"var(--cbp-border-radius-soft)",
+                          "border-bottom-right-radius":"var(--cbp-border-radius-soft)",
+                          "border-width":"2px",
+                          "border-style":"solid",
+                          "border-color":"var(-cbp-color-gray-cool-40)",
+                          "border-top":"none",
+                          "color":"var(--cbp-structured-list-color)",
+                          "font-size":"var(--cbp-font-size-masthead-1)",
+                          "font-weight":"var(--cbp-font-weight-black)",
+                          "line-height":"1.75"
```

Comment:
token?

Reply:
unitless values for line height don't have vars, discussed on call to confirm